### PR TITLE
add check mark icon with system vibrate when scan image

### PR DESCRIPTION
**Pull request by dipang90**
change the ReviewViewController. 
Add the check mark icon with system vibrate. When image has been scanned, the system will be vibrate and green check mark will be add.

### CODE REVIEW

**Comment by BasThomas** [comment]
Hi @dipang90, thanks for your contribution!

Can you give some more information about this change? Perhaps post a video demonstrating the change?

**Comment by BasThomas** [comment]
Hi @dipang90, thanks for your contribution!

Can you give some more information about this change? Perhaps post a video demonstrating the change?

File: WeScan/Review/ReviewViewController.swift [hunk 1]
```diff
@@ -15,6 +16,15 @@ final class ReviewViewController: UIViewController {
     private var enhancedImageIsAvailable = false
     private var isCurrentlyDisplayingEnhancedImage = false
     
+    lazy var imageCheck: UIImageView = {
+        let imageView = UIImageView()
+        imageView.image = UIImage(named: "img_check")
+        imageView.backgroundColor = UIColor(red: 1, green: 1, blue: 1, alpha: 0.0)
```
Comment (BasThomas) [inline WeScan/Review/ReviewViewController.swift:22]: To keep dark mode in mind, could we use a semantic color instead? Or if a specific color is required, use a named version?
```suggestion
        imageView.backgroundColor = .black
```

File: WeScan/Review/ReviewViewController.swift [hunk 2]
```diff
@@ -90,6 +104,8 @@ final class ReviewViewController: UIViewController {
     
     private func setupViews() {
         view.addSubview(imageView)
+        imageView.addSubview(imageCheck)
+        AudioServicesPlaySystemSound(kSystemSoundID_Vibrate)
```
Comment (BasThomas) [inline WeScan/Review/ReviewViewController.swift:108]: Why did you opt to use `AudioToolbox` over using haptics here? Could we instead consider using `UINotificationFeedbackGenerator`?

File: WeScanSampleProject/Assets.xcassets/img_check.imageset/Contents.json [hunk 3]
```diff
@@ -0,0 +1,21 @@
+{
```
Comment (BasThomas) [inline WeScanSampleProject/Assets.xcassets/img_check.imageset/Contents.json:1]: Does this look OK/as expected in both light and dark mode?

**Review by kairadiagne — CHANGES_REQUESTED** [review]
Awaiting feebdack on @BasThomas comments.

**Pull request closed without merging**